addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -19,153 +17,241 @@
-            <ul class="nav">
-                <iframe t-attf-src="/google_map?width=260&amp;height=240&amp;partner_ids=#{ google_map_partner_ids }&amp;partner_url=/partners/"
-                    style="width:260px; height:240px; border:0; padding:0; margin:0;" scrolling="no"></iframe>
-            </ul>
+            <div class="btn-group ms-2">
+                <button class="btn btn-light border-primary active">
+                    <i class="fa fa-th-large"/>
+                </button>
+                <button class="btn btn-light" data-bs-toggle="modal" data-bs-target=".partner_map_modal">
+                    <i class="fa fa-map-marker" role="img" aria-label="Open map" title="Open map"/>
+                </button>
+            </div>
         </t>
     </xpath>
 </template>
 
+<template id="o_wcrm_partner_address" inherit_id="website_crm_partner_assign.index" name="Address">
+    <xpath expr="//small[@id='o_wcrm_partners_address']" position="inside">
+        <div t-field="partner.self" 
+            t-options="{
+                'widget': 'contact',
+                'fields': ['country_id']
+            }"
+            class="py-2"
+        />
+    </xpath>
+</template>
+
 <template id="partner" name="Partner Detail">
     <t t-call="website_crm_partner_assign.layout">
-        <div class="col-lg-5">
-            <ol t-if="not edit_page" class="breadcrumb">
-                <li class="breadcrumb-item"><a t-attf-href="/partners#{current_grade and '/grade/%s' % slug(current_grade)}#{current_country and '/country/%s' % slug(current_country)}">Our Partners</a></li>
-                <li class="breadcrumb-item active"><span t-field="partner.display_name"/></li>
-            </ol>
+        <div t-if="not edit_page" class="mb-3">
+            <a t-attf-href="/partners#{current_grade and '/grade/%s' % slug(current_grade)}#{current_country and '/country/%s' % slug(current_country)}" aria-label="Back to resellers list"><i class="fa fa-chevron-left me-2"/>Back to resellers</a>
         </div>
         <t t-call="website_partner.partner_detail">
             <t t-set="right_column">
-                <div id="right_column" class="mb16"><t t-call="website_crm_partner_assign.references_block"/></div>
+                <div id="right_column"><t t-call="website_crm_partner_assign.references_block"/></div>
             </t>
         </t>
     </t>
 </template>
 
 <template id="grade_in_detail" inherit_id="website_partner.partner_detail">
-  <xpath expr="//*[@id='partner_name']" position="after">
-    <h3 class="col-lg-12 text-center text-muted" t-if="partner.grade_id and partner.grade_id.website_published">
-      <span t-field="partner.grade_id"/></h3>
-  </xpath>
+    <xpath expr="//*[@id='partner_name']" position="before">
+        <span class="col-lg-12 text-muted" t-if="partner.grade_id and partner.grade_id.website_published">
+            <span t-field="partner.grade_id"/> Reseller
```

Comment:
I removed the "Reseller" part, I think it will work fine this way ! 

Thanks a lot for the review. I pushed the latest changes, I think we are good now 🚀

Reply:
@thiryr Btw, I don't have merging rights so I will need your help for that part (if you do have them ofc)

Reply:
Me neither, as it so happens :D
I think @stefanorigano is a reviewer for your team? Otherwise ping tde I guess